vnext/DeforkingPatches/.clang-format

```diff
@@ -1,91 +1,4 @@
 ---
```

Comment:
We previously had a no-op clang-format in the Yoga dir, and this real one in the top level dir. I moved the no-op one to the top level since we discovered FB doesn't CI enforce formatting.